test/index.js

```diff
@@ -115,8 +114,6 @@ test('Grunt-localizr', function (t) {
             t.end();
         });
     });
-    t.on('end', function() {
-        process.chdir(original);
```

Comment:
I think the chdir was only necessary because of `process.cwd()` to resolve the path to fixtures. removed dependence on `process.cwd()` entirely